src/devices/cpu/h8/h8_sci.h

```diff
@@ -112,7 +112,8 @@ class h8_sci_device : public device_t {
 
 	int eri_int, rxi_int, txi_int, tei_int;
 
-	int tx_state, rx_state, tx_bit, rx_bit, clock_state, clock_mode, tx_parity, rx_parity, ext_clock_counter;
+	int tx_state, rx_state, tx_bit, rx_bit, clock_state, tx_parity, rx_parity, ext_clock_counter;
+	ClockMode clock_mode;
```

Comment:
I noticed a bug concerning save states `clock_mode` isn’t saved in save state, which is OK as it’s entirely derived from other members.  However, that means it needs a post load callback to call `clock_update`, or it might get out-of-sync.

Reply:
I can't see how to do that sorry. Can you point me at an example please?

Reply:
> I can't see how to do that sorry. Can you point me at an example please?

Basically, just override `void device_post_load()` (inherited from `device_t`) and call `clock_update()` – that should be all that’s necessary.

Reply:
Done.
I noticed the code base seems to use `virtual` and `override` on the same function def quite a bit.
Any idea if this is by design?
Is it OK to follow this: https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c128-virtual-functions-should-specify-exactly-one-of-virtual-override-or-final